src/app/components/Header/ScriptLink/index.tsx

```diff
@@ -57,31 +69,33 @@ const ScriptLinkContainer = ({ scriptSwitchId = '' }) => {
   if (!scriptLinkEnabled || isNextJs) return null;
 
   // eslint-disable-next-line react-hooks/rules-of-hooks
-  const { path, params } = useRouteMatch();
+  const { path, params }: UseRouteMatcher = useRouteMatch();
```

Comment:
Yeah I can fix, just need to move it before the return statement, don't think this would've actually done anything, just the linter being super strict